pkg/prometheus_exporter/aggregating_counter.go

```diff
@@ -2,25 +2,27 @@ package prometheus_exporter
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
-	"github.com/sirupsen/logrus"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
Why `dto`?

Reply:
https://github.com/prometheus/client_golang/blob/master/prometheus/examples_test.go#L29

Mainly because I just copied it from the example, but.. well it is `dto` more or less so didn't event think about it.
Do you want to change it?

Reply:
Not really, was just curious what `dto` actually means.